src/playground-file-editor.ts

```diff
@@ -210,7 +210,7 @@ const mimeTypeToTypeEnum = (mimeType?: string) => {
   if (encodingSepIndex !== -1) {
     mimeType = mimeType.substring(0, encodingSepIndex);
   }
-  switch (mimeType.toLowerCase()) {
```

Comment:
I think we should keep this normalization too -- because the `Content-Type` can also be set directly in the `project.json` file.

(Actually, do you know why this wasn't sufficient? There must be another place where we check the `Content-Type` that you were hitting?)

Reply:
I believe this was sufficient. Just moved it earlier in case the `content-type` header gets used elsewhere.